src/client/components/Header/Header.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import './Header.less';
+import LocationPicker from './../LocationPicker/LocationPicker';
```

Comment:
eslint error

the rule: https://github.com/benmosher/eslint-plugin-import/blob/v2.16.0/docs/rules/no-useless-path-segments.md

also fix your configs so that this error shows up for you

```suggestion
import LocationPicker from '../LocationPicker/LocationPicker';
```

Reply:
done